modules/openvpn_status/openvpn_status.go

```diff
@@ -0,0 +1,96 @@
+package openvpn_status
+
+import (
+	"os"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"github.com/netdata/go.d.plugin/pkg/matcher"
+)
+
+const (
+	defaultFilePath = "/var/log/openvpn/status.log"
+)
+
+func init() {
+	creator := module.Creator{
+		Defaults: module.Defaults{
+			Disabled: true,
```

Comment:
Why disabled?

Reply:
Until we remove python collector with same name?

Reply:
The python one name is ovpn_status_log. Ideally, we should name the go version ovpn_status_log too (but I think it is ok to not do it): then only one of them (py/go) can be active at the same time.